ethers-providers/src/lib.rs

```diff
@@ -561,3 +561,17 @@ pub trait Middleware: Sync + Send + Debug {
             .map_err(FromErr::from)
     }
 }
+
+#[cfg(feature = "celo")]
+#[async_trait]
+pub trait CeloMiddleware: Middleware {
+    async fn get_validators_bls_public_keys(
+        &self,
```

Comment:
This should probably be an `Option<BlockId>`? https://github.com/gakonst/ethers-rs/blob/b4f490844b5d81c40038b2c6383b0292cc8c99ed/ethers-providers/src/lib.rs#L252